### PR TITLE
Added cleanup for old RRD files to daily.sh

### LINKED ISSUE #2388: Housekeeping in Librenms
First of all hats off to all the developers who make this tool so great ! 
It would be very helpful for an environment with 500+ hosts, if there is an utility php to 
1. Cleanup old / never used rrd files 
2. Stale database entries 
3. Clean up old alert,syslog and event logs according to the age .
Just an optional things but it will be become very handy in long run  :) :+1:

### PR DESCRIPTION
**Pull request by laf**
Fix #2388

### CODE REVIEW

**Comment by f0o** [comment]
:+1:

**Comment by paulgear** [comment]
My thoughts:

1. If it's set to 0, it should skip the check. I think if it is run with the setting at 0 at the moment, it will delete all RRD files.

1. Once the above is fixed, the config.php.default setting should be 0.  It's easy to clean up data you don't care about if you run out of space, but it's impossible to recreate data you do care about but has already been deleted.

1. The comment should be consistent with the others in config.php.default (before the check, and # style).

**Comment by laf** [comment]
Done

File: daily.php [hunk 1]
```diff
@@ -31,6 +31,14 @@
     exit(0);
 }
 
+if ($options['f'] === 'rrd_purge') {
+    if (is_numeric($config['rrd_purge']) && $config['rrd_purge'] > 0) {
+        $cmd = "find ".$config['rrd_dir']." -mtime +".$config['rrd_purge']." -exec rm -Rf {} \;";
+        $purge = `$cmd`;
+        echo 'purged old rrd files';
```
Comment (paulgear) [inline daily.php]: Might be nice to echo $purge here so people can know what was changed.

File: daily.php [hunk 2]
```diff
@@ -31,6 +31,16 @@
     exit(0);
 }
 
+if ($options['f'] === 'rrd_purge') {
+    if (is_numeric($config['rrd_purge']) && $config['rrd_purge'] > 0) {
+        $cmd = "find ".$config['rrd_dir']." -mtime +".$config['rrd_purge']." -exec rm -Rf {} \;";
```
Comment (murrant) [inline daily.php]: This doesn't print output, you'll need to add -print in the find command

Comment (laf) [inline daily.php]: Done.

**Comment by murrant** [comment]
👍 from me, but do you think we need to mention it in the docs, or is the line in config.php.default enough?

**Comment by laf** [comment]
I've added a bit more to try and be clearer but yes we should mention as much as possible in docs to save queries.

**Pull request merged**